apache/ports.conf.in

```diff
@@ -0,0 +1,9 @@
+Listen ${APACHE_PORT}
+
+<IfModule mod_ssl.c>
+    Listen 443
```

Comment:
SSL offloading for services running on k8s will be done on cloudfront level, so service should listen on port 80